### PR TITLE
WIP: NMMA analysis service

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR creates an NMMA analysis service demo.

Blocked by https://git.ligo.org/lscsoft/bilby/-/merge_requests/1043.

### CODE REVIEW

File: services/nmma_analysis_service/app.py [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+import requests
+
+from tornado.ioloop import IOLoop
+import tornado.web
+import tornado.escape
+
+from astropy.time import Time
+from astropy.table import Table
+import bilby
+from nmma.em.analysis import get_parser, main
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply services/nmma_analysis_service/app.py:21]: Good call.

**Comment by mcoughlin** [comment]
@profjsb Do we have the ability to turn on and off the sn_analysis_service from the config? Does it start-up without a supervisor script?

**Comment by profjsb** [comment]
Since this is a highly specialized (and computationally expensive) codebase, I believe that it should be its own standalone repo (e.g. in https://github.com/skyportal-contrib) or as a skyportal extension in Fritz (ie. inside `services/` in  https://github.com/fritz-marshal/fritz/tree/main/extensions/skyportal)

**Comment by profjsb** [comment]
Let's open this up at Fritz-marshal as an extension.

**Pull request closed without merging**